src/components/email-sign-up/component.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { TextField, Button } from "@material-ui/core";
+
+class EmailSignUp extends Component {
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+    onFormSubmit: PropTypes.func.isRequired
+  };
+
+  state = {
+    email: ""
+  };
+
+  handleInputChange = this.handleInputChange.bind(this);
```

Comment:
Are we going to skip the @bound decorator in the environment?

Reply:
@Robbieee Yes, it caused a lot of trouble when we were testing it in web